server/wfone-notification-push-api/wfone-notification-push-service-api/src/main/java/ca/bc/gov/nrs/wfone/notification/push/service/api/v1/spring/ServiceApiSpringConfig.java

```diff
@@ -114,14 +114,17 @@ public MonitorHandler spatialMonitorHandler() {
 	@Value("${WFONE_FIREBASE_DB_URL}")
 	private String firebaseDbUrl;
 
+	@Value("${FIREBASE_CONFIG_JSON}")
+	private String firebaseConfigJson;
+
 	@Bean
 	FirebaseMessaging getFireBaseMessenger() {
 		FirebaseMessaging result;
 
 		try {
 
 			String[] scopes = { "https://www.googleapis.com/auth/firebase.messaging" };
-			Resource resource = new ClassPathResource("firebase-config.json");
+			Resource resource = new ClassPathResource(firebaseConfigJson);
```

Comment:
Resource resource = new ByteArrayResource(firebaseConfigJson.getBytes());

Reply:
I think we need to use a ByteArrayResource to accept the actual value from the variable